src/App.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  Card, CardImg, CardText, CardBody, CardLink,
+  CardTitle, CardSubtitle
+} from 'reactstrap';
+import './App.css';
+
+
+
+import {applications} from './shared/applications'
+
+const jobapps = applications;
```

Comment:
since you've imported applications you shouldn't have to save it to a new variable - jobapps.
you could just work directly with the import